client-js/app/elements/selectable-table.html

```diff
@@ -15,7 +15,7 @@
       is: 'selectable-table',
       extends: 'tbody',
       behaviors: [
-        Polymer.IronSelectableBehavior
+        Polymer.IronMultiSelectableBehavior
```

Comment:
Thanks for picking this up, I was going to add multi-selectable to this PR, but figured I do this one thing at a time.